.github/workflows/cd.yml

```diff
@@ -197,15 +197,6 @@ jobs:
           tag: "template-tag-list"
           allowUpdates: true
 
-      - name: download simpleauth to fx-core
-        uses: nick-invision/retry@v2
-        with:
-          timeout_minutes: 10
-          max_attempts: 10
-          retry_on: error
-          command: |
-            bash ./.github/scripts/download-simpleauth.sh
```

Comment:
I don't know about the detail of the github actions, but. Does this script attempt to run a file that was just deleted?

Reply:
V5 will never use simpleauth, so no need to download simpleauth.